Dockerfile.slim

```diff
@@ -23,7 +23,7 @@ RUN apk update \
     openssh-client
 
 # AWS CLI
-RUN pip install --upgrade awscli==${AWS_CLI_VERSION} s3cmd==${AWS_CLI_S3_CMD} python-magic \
+RUN pip install --upgrade  --no-build-isolation awscli==${AWS_CLI_VERSION} s3cmd==${AWS_CLI_S3_CMD} python-magic \
```

Comment:
What does --no-build-isolation do?

Reply:
From what I understand, it prevents some automatic install of some dependencies:  
https://stackoverflow.com/questions/62889093/what-does-no-build-isolation-do/62889267#62889267

This was the recommended flag from AWS to get the CLI install working.  Whether it's "right" is a different question. 
 I'll bet testing more shortly BUT getting this out here now while I do that testing.

Reply:
Additional notes:  this seems to be dependent upon which version of python & libs.  AKA builds failed on ubuntu.  Ubuntu  bionic is out of standard support so this MAY not be required UNTIl we start other upgrades.

Reply:
What breaks is the pip install.  Will NOT impact any EXISTING built images and It's PRIMARILY impacting Alpine builds - started in July? due to a dependency of the aws cli pip install.   Bionic may not have the issue but bionic is WAY out of date, testing that as well in this.  Will LIKELY revert the bionic and do a separate PR with JUST the ubuntu changes.